faq/cohort-subs-guide.rst

```diff
@@ -0,0 +1,149 @@
+Before starting a submission, we strongly advise you to contact us at cohort-dataflow@ebi.ac.uk if you are planning to submit a
+linked cohort dataset, including some details about your study, and we can give guidance on your sample structure, and how
+to complete the data submissions. The Samples structure will be be created in the `BioSamples Archive <https://www.ebi.ac.uk/biosamples/>`_
+, which is a resource at the EBI (European Bioinformatics Institute).
+
+1. Create the top-level Sample
+''''''''''''''''''''''''''''''
+
+The first step is to create top-level Samples. These Samples will represent each case or patient in the study.
+This is represented by Sample #1 in the diagram. If this is a human sample, this can contain minimal, non-identifying metadata about
+the patient (e.g. gender, organism, disease). See an example `here <https://www.ebi.ac.uk/biosamples/samples/SAMEA12928716>`_.
+
+Top-level Sample records can be created in BioSamples using the `BioSamples uploader tool <https://www.ebi.ac.uk/biosamples/docs/cookbook/upload_files>`_.
+
+.. note ::
+    The ENA and the `EGA (European Genome Phenome Archive) <https://ega-archive.org/>`_ are the only archives which integrate
+    BioSample records into their :doc:`database structure <submit/general-guide/metadata>`. For data deposited at other
+    archives, additional BioSample records may need to be created (in BioSamples) to represent those data.
+
+2. Submit Pathogen Sequence data to the ENA
+'''''''''''''''''''''''''''''''''''''''''''
+
+The next step is to submit your pathogen nucleotide records (raw reads or assembly data) to the ENA.
+The :doc:`Pathogen Submissions Guide <faq/pathogen-subs-guide>` provides a quick introduction to the ENA and tips for
+Pathogen data submissions.
+
+
+2. Create the child Samples in additional database resources
+''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+The next step is to create your datasets in the correct database for the data type. The `EBI submissions wizard
+<https://www.ebi.ac.uk/submission/>`_ can help direct you to a resource to deposit your data.
+We can reccommend the following database resources for common data types:
+
+- For sensitive human nucleotide records and human clinical epidemiological data which requires controlled access, please
+  contact the `EGA (European Genome Phenome Archive) <https://ega-archive.org/>`_ to start a submission.
+- For expression data, or uncategorsied datasets, please use `ArrayExpress/BioStudies <https://www.ebi.ac.uk/biostudies/arrayexpress>`_
+
+After the datasets have been submitted in the appropriate databases, the required child Samples for linking can be created.
+The child samples will represent their relationship to the top-level Sample. Different samples can be used for different
+data types **and** for different time points. Please contact us if you have any doubts about setting up your sample structure.
+
+
```

Comment:
I would split this out into 2 sections for clarity - e.g.:
2. Submit other data types to appropriate database resources
3. Create the child samples for linking

Reply:
step 2 is now split into steps 3 and 4